### PR TITLE
Add the files of exp1, 2, 3

### PR DESCRIPTION
**Pull request by Fazail333**
Added the complete codes of lab 1 and lab 3. lab 2 is still pending , I will try to complete lab 2 in the next week.

### CODE REVIEW

**Review by UmerShahidengr — APPROVED** [review]
Files are complete. LGTG

**Review by UmerShahidengr — APPROVED** [review]
Ready to be merged now

File: Fazail/.gitignore
[the remark this answers is not in the document]
Comment (Fazail333) [reply Fazail/.gitignore:1]: Thanks.

File: Fazail/assignments/16-bit-comb-signed-multiplier/.gitignore
[the remark this answers is not in the document]
Comment (Fazail333) [reply Fazail/assignments/16-bit-comb-signed-multiplier/.gitignore:1]: Ok

File: Fazail/assignments/16-bit-comb-signed-multiplier/Makefile
[the remark this answers is not in the document]
Comment (Fazail333) [reply Fazail/assignments/16-bit-comb-signed-multiplier/Makefile:1]: Thank you Sir .

File: Hamza/README.md
[the remark this answers is not in the document]
Comment (Fazail333) [reply Hamza/README.md:1]: Sir it was very first commit. I **mistakenly**  open the Hamza's README.md file and edit that file .Before commit I didn't notice which files are going to commit. Then I see on the github that commits changed both hamza's file and mine. I tried to delete the commit but I am not able to delete that commit.

So, by mistake I changed that file. Next time I try not to do this mistake again
 
Is there any way to delete such kind of commits ?

**Pull request merged**